pkg/cassandra/encryption.go

```diff
@@ -164,6 +164,11 @@ func ClientEncryptionEnabled(template *DatacenterConfig) bool {
 }
 
 func ServerEncryptionEnabled(template *DatacenterConfig) bool {
+	// This allows to bypass the automated behavior, but of course removes the idea of validating this field also
+	if template.ServerEncryptionStores == nil {
```

Comment:
We can (for consistency), but are we going to have per-pod stuff there?

Reply:
We're going to have to inject them nonetheless at some point rather than read secrets directly from the operator, even if they are the same for all pods.

Reply:
but this is an optional suggestion, non blocking for this PR.

Reply:
I already solved this in the PR.